### DISCUSSION TITLE
Dịch `Mode Collapse` trong GAN

### DISCUSSION THREAD

**Discussion by quangnhat185** [mlbvn/handson-ml2-vn#165] [Terminology] [answered]
Từ `Mode Collapse` trong GAN đề cập đến hiệng tương Generator chỉ tạo ra một tập các dữ liệu nhỏ khá giống nhau bởi Discriminator không thể phân biệt được các mẫu này. 

Mọi người suggest cho cách dịch cụm từ này nhé?

cc: @mlbvn/handson-ml

**Comment by cuongvng** [comment, marked as answer]
em đang dịch là `sụp đổ mode`.

**Reply by minhduc0711** [reply]
từ mode này có phải là từ mode trong thống kê không nhỉ?

**Reply by cuongvng** [reply]
em nghĩ là không.

**Reply by lkhphuc** [reply]
Mình tưởng đúng là mode trong mean, mode mà. Giống như 1 multivariate gaussian distribution có 2 cái đỉnh (mode) thì GAN nó có xu hướng chỉ dồn hết probability mass vào mô phỏng 1 cái đỉnh, còn không có probability nào ở cái mode còn lại nên hay mới gọi là mode collapse.

**Reply by minhduc0711** [reply]
Mình xem thử [bài báo này](https://proceedings.neurips.cc/paper/2017/file/44a2e0804995faf8d2e3b084a1e2db1d-Paper.pdf) thì có vẻ đúng
![image](https://user-images.githubusercontent.com/34294427/177991432-bc99ae63-05cc-43cc-9fb9-ceef92002ee3.png)
